src/components/playback/playbackmanager.js

```diff
@@ -3811,7 +3819,7 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
         });
     };
 
-    PlaybackManager.prototype.shuffle = function (shuffleItem, player, queryOptions) {
```

Comment:
Did a bit of cleaning as well in ``playbackManager``. This was removed because it's unused in the function.